### ISSUE TITLE
[feature request] Export to pgfgantt

### ISSUE THREAD

**Issue by mxa** [bardsoftware/ganttproject#2298]
Ganttproject is good to manage projects, I wish I could easily export it to [pgfgantt](https://www.ctan.org/pkg/pgfgantt) for typesetting.

**Comment by dbarashev** [comment]
What is the purpose? Embed a chart image into PDF docs?

**Comment by mxa** [comment]
Project proposals for example.

**Comment by dbarashev** [comment]
I mean, technically what is the purpose? If you want to embed a chart, you can use PDF export. It produces a vector image.

**Comment by mxa** [comment]
No. That's not sufficient. pgfgantt can create really nice looking graphs in the layout of your document, it's not the same as a generic vector output.
